common/src/main/java/org/jboss/hal/testsuite/fragment/shared/util/ResourceManager.java

```diff
@@ -11,26 +12,58 @@
 import org.jboss.hal.testsuite.util.Console;
 import org.jboss.hal.testsuite.util.PropUtils;
 import org.openqa.selenium.By;
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.util.List;
+import java.util.concurrent.TimeUnit;
 
 /**
  * @author jcechace
  */
```

Comment:
I suggest to remove this, as git blame tells you more and this is becoming outdated.